internal/controller/controllers/agent_controller.go

```diff
@@ -1250,6 +1250,32 @@ func (r *AgentReconciler) updateHostIgnition(ctx context.Context, log logrus.Fie
 	return err
 }
 
+func (r *AgentReconciler) updateNodeLabels(log logrus.FieldLogger, host *common.Host, agent *aiv1beta1.Agent, params *installer.V2UpdateHostParams) (bool, error) {
```

Comment:
I assume if we call agent -> host lets change it to host labels?

Reply:
To be exact, this function just modifies the update params. It doesn't modify the host here.